src/main/java/me/totalfreedom/totalfreedommod/command/Command_wiperegions.java

```diff
@@ -32,21 +32,21 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
         World world = server.getWorld(args[0]);
         if (world == null)
         {
-            msg("There is no world named \"" + args[0] + "\"", ChatColor.RED);
+            msg("There is no world named \"" + args[0] + "\".", ChatColor.RED);
```

Comment:
Again full stops after variables.

Reply:
I think its fine because its ". but i am not sure

Reply:
its not, fixed